git_guide.md

```diff
@@ -0,0 +1,22 @@
+# Git guide
```

Comment:
Can you link to this from the beginner's guides in `CONTRIBUTING_code`? https://github.com/mozilla-mobile/shared-docs/blob/master/android/CONTRIBUTING_code.md#beginners-guides

Reply:
provide a link in beginners-guides.